src/components/ExplorerIntroNational.js

```diff
@@ -1,7 +1,7 @@
 import PropTypes from 'prop-types'
 import React from 'react'
 
-import { nibrsTerm, srsTerm } from './Terms'
+import { estimatedTerm, nibrsTerm, srsTerm } from './Terms'
```

Comment:
@RyanSibley We first want to pull in the estimated term from our `Terms` file. This isn't all terms, but just a few of the ones we use all the time (estimated, NIBRS, and summary at the moment).